openfl-workspace/keras_nlp/src/dataloader_utils.py

```diff
@@ -181,7 +189,10 @@ def get_datasets_(encoder_input_data, decoder_input_data,
     return results
 
 
-def load_shard(collaborator_count, shard_num, data_path, num_samples, split_ratio):
+def load_shard(
+    collaborator_count: int, shard_num: Any, data_path: str,
```

Comment:
```suggestion
    collaborator_count: int, shard_num: int, data_path: str,
```

Reply:
So we can remove `shard_num = int(shard_num)` line.
But we have to cast the value to int here https://github.com/intel/openfl/blob/fef41fb874a092e97dfa532f53fdae7bf3dcb1a5/openfl-workspace/keras_nlp/src/nlp_dataloader.py#L35